app/.gitignore

```diff
@@ -0,0 +1 @@
+/build
```

Comment:
Why do we need one .gitignore for app/ and one .gitignore for the root?

Reply:
We don't. This is generated by IntelliJ, but I agree it's unnecessary. Cleaned up.